packages/TelegramClient-UI.package/TCUNotification.class/instance/shrinkToContent.st

```diff
@@ -0,0 +1,8 @@
+drawing
+shrinkToContent
```

Comment:
I think it should rather be calles `shrinkContent` as the content gets shinked here instead of the Notification being shrinked to the contents size.

Reply:
This makes sense, thank you